portfolio/src/main/webapp/style.css

```diff
@@ -242,6 +247,6 @@ p {
   height: 20px;
   border-radius: 50%;
   position: absolute;
-  left: 35.6%;
+  left: 35.1%;
```

Comment:
what is this magic number? :)

Reply:
It's used to align this circle onto the line 
![Screenshot 2020-05-29 at 12 02 14 PM](https://user-images.githubusercontent.com/22436066/83314187-238d0200-a1ce-11ea-84e0-a3d1effa7fd7.png)
Even one percentage point moves the center off the line and makes it look strange :)

Reply:
CSS is hard :) Does it still look nice when you resize the window?